lib/ret_web/controllers/page_controller.ex

```diff
@@ -200,13 +200,18 @@ defmodule RetWeb.PageController do
 
   def render_hub_content(conn, hub, _slug) do
     hub = hub |> Repo.preload(scene: [:screenshot_owned_file], scene_listing: [:screenshot_owned_file])
+
     {app_config_script, app_config_csp} = generate_app_config()
 
+    {available_integrations_script, available_integrations_csp} =
```

Comment:
Ah, I was assuming we'd put this into the `/api/v1/meta` API, is there a reason injecting it at page load time seems better or worse? i guess since we already have the pattern for configs, these are kind of equivalent in terms of impact, so i'm learning towards your approach.

Reply:
Just felt right to guarantee this information was available before the rest of the javascript run, without having to wait for another request. I don't see a strong reason to put in in the API for this purpose, but we can always also put it there if we need it later.